src/python/pants/option/global_options.py

```diff
@@ -276,8 +273,7 @@ def register_bootstrap_options(cls, register):
              default=DEFAULT_EXECUTION_OPTIONS.remote_store_chunk_bytes,
              help='Size in bytes of chunks transferred to/from the remote file store.')
     register('--remote-store-chunk-upload-timeout-seconds', type=int, advanced=True,
-             default=DEFAULT_EXECUTION_OPTIONS.remote_store_chunk_upload_timeout_seconds,
-             help='Timeout (in seconds) for uploads of individual chunks to the remote file store.')
+             default=0, help='Deprecated and ignored.')
```

Comment:
Would be good to include a target removal date here, so that it's more obvious when it can actually be removed.

Reply:
Kind of tempted to just remove the flag, as I'd be surprised if anyone was actually setting it... Any strong objection? :)